config_files/config_de_novo_uppmax.yaml

```diff
@@ -1,7 +1,7 @@
 Pipelines:
  QCcontrol: ["fastqc", "abyss", "trimmomatic", "kmergenie", "align"]
  assemble: ["abyss", "allpaths", "cabog",  "masurca", "soapdenovo", "spades", "trinity"]
- evaluete: ["align", "qaTools", "FRC"]
+ evaluete: ["align", "qaTools", "FRC", "BUSCObacteria", "BUSCOeukarya", "BUSCOvertebrata", "BUSCOfungi"]
  align: []
```

Comment:
Basically I wanted the path to the datasets somewhere in the global config file, and not hardcode anything in the pipeline script. The dataset directories are downloaded manually in separate tarballs, so I didn't want to make any assumptions that they are contained in the same path, ie. `/foo/bar/[bacteria / vertebrata / etc]` Only alternative I see is to do something like

```YAML
BUSCO:
  bin: /foo/BUSCO
  options: [...]
  bacteria: /foo/bacteria/
  vertebrata: /bar/vertebrata/
```